packages/api-page-builder/src/plugins/models/pbCategory.model.ts

```diff
@@ -6,6 +6,16 @@ import { withFields, string, withName } from "@webiny/commodo";
 export default ({ createBase }) =>
     flow(
         withName("PbCategory"),
+        withHooks({
+            async beforeDelete() {
+                const Model = context.models[this.modelId];
```

Comment:
You can see all of the available models here `packages/api-page-builder/src/plugins/models.ts`

So, change `const Model = context.models[this.modelId];` to `const { PbPage } = context.models;` and do a `findOne` with the `PbPage` model below.

Reply:
Btw, `withHooks` needs to be imported as well.